package.json

```diff
@@ -350,6 +360,17 @@
                     "when": "clangd.ast.hasData"
                 }
             ]
-        }
+        },
+        "colors": [
```

Comment:
nit: can we move this section above to the `commands` section? so that all inactiveRegion settings are closer.